lib/sheldon/brain.rb

```diff
@@ -0,0 +1,32 @@
+class Brain
+
+  def learn(recall_cue, abs_learn_path)
+    synapse = find_synapse(recall_cue)
+    basename = File.basename(abs_learn_path)
+    FileUtils.mkdir_p(synapse)
+    FileUtils.mv(abs_learn_path, synapse)
+  end
+
+  def recall(recall_cue, destination)
+    synapse = find_synapse(recall_cue)
+    FileUtils.ln_s(read_synapse(recall_cue), destination)
+  end
+
+  private
+
+  def find_self
+    relative_path = '~/sheldon2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.